package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
```

Comment:
From README: "We only use Yarn as our official Node package manager, and so we will only commit yarn.lock files, and ignore package-lock.json files."
Please remove package-lock.json from this PR and add it to the .gitignore. Yarn is much more efficient than vanilla npm.

Reply:
okay